### ISSUE TITLE
[React Native Gallery-> Expander]: Visual looking heading “Expander” is not defined as heading.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#305]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to any button (E.g., Expander) and invoke it.
Set narrator to headings mode using caps lock + Page up/down.
Now navigate to “Expander” heading and listen to the narrator’s announcement.
**Actual Result:**
Visual looking heading “Expander” is not defined as heading. When user navigates to this heading narrator simply announcing as static text like “Expander”.

Note: Same issue is observed in entire application.

**Expected Result:**
Visual heading “Expander” should be defined as the heading. When user navigates to this heading narrator should announce as ““Expander, heading level 1”.

**User Impact:**
Users with visual impairments who rely on screen reader may use heading navigation mode to render through the page. Not defining heading in the application will make it difficult for users navigating in the app.

Internal bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/43361294

**Comment by AgneLukoseviciute** [comment]
We added support for headings in August (https://github.com/microsoft/react-native-gallery/issues/306).
